data/ccEnvironments.json

```diff
@@ -0,0 +1,18 @@
+[
```

Comment:
Is this file necessary?

Reply:
I'll look into it, and make adjustments if necessary in subsequent PRs.